core/src/main/java/org/infinispan/functional/impl/MetaParamsInternalMetadata.java

```diff
@@ -113,60 +121,32 @@ NumericVersion getNumericVersion() {
       return version instanceof NumericVersion ? (NumericVersion) version : null;
    }
 
-   void setNumericVersion(NumericVersion version) {
-      params.add(new MetaEntryVersion(version));
-   }
-
    @ProtoField(number = 2)
    SimpleClusteredVersion getClusteredVersion() {
       EntryVersion version = version();
       return version instanceof SimpleClusteredVersion ? (SimpleClusteredVersion) version : null;
    }
 
-   void setClusteredVersion(SimpleClusteredVersion version) {
-      params.add(new MetaEntryVersion(version));
-   }
-
    @ProtoField(number = 3, defaultValue = "-1")
```

Comment:
Not sure -1 is a proper default here. the implementation of ```created()``` method indicates 0 is the default.  But tbh I have no idea what 0 means for a timestamp. Is it the epoch? Is it Noah's birthday?